test/e2e-common.sh

```diff
@@ -190,15 +189,10 @@ function install_knative_serving_standard() {
     kubectl autoscale -n istio-system deploy istio-pilot --min=3 --max=10 --cpu-percent=60 || return 1
   fi
 
-  wait_until_pods_running knative-serving || return 1
-  wait_until_pods_running istio-system || return 1
-  wait_until_service_has_external_ip istio-system istio-ingressgateway
-
   if [[ -n "${INSTALL_MONITORING_YAML}" ]]; then
     echo ">> Installing Monitoring"
     echo "Knative Monitoring YAML: ${INSTALL_MONITORING_YAML}"
     kubectl apply -f "${INSTALL_MONITORING_YAML}" || return 1
-    wait_until_pods_running knative-monitoring || return 1
```

Comment:
Should we keep the checks here as well? This will alter the notion of these functions to not be "complete" once they finish.

Reply:
If we keep the checks here, it will defeat the split done by this PR. Note that this particular check is now done at line 251.